### DISCUSSION TITLE
How to not import react in every jsx, js file?

### DISCUSSION THREAD

**Discussion by crapthings** [meteor/meteor#12528] [Feature Requests]
* ~~use script tag~~

* ~~import React from 'react'~~

```
import React from 'react'
window.React = React
```

is there any other options?

<img width="1555" alt="image" src="https://user-images.githubusercontent.com/1147704/221766974-4a902ca8-aac0-4b6b-9020-084518facf86.png">

**Comment by StorytellerCZ** [comment]
I don't think this is the proper sport for this question.

Anyhow, here is one potential answer to your question: https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

**Reply by henriquealbert** [reply]
yep! you can use `.babelrc` file and add this code:

````
{
  "plugins": [
    ["@babel/plugin-transform-react-jsx", {
      "runtime": "automatic"
    }]
  ]
}
````

**Reply by henriquealbert** [reply]
"@babel/preset-react" will not work because meteor does not let you override the configuration
